us.materialio.Materialious.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id type="desktop-application">us.materialio.Materialious</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Materialious</name>
+  <summary>Materialious is a privacy respecting frontend for YouTube built ontop of Invidious.</summary>
+  <description>
+    <p>Materialious is a client for Invidious using material design. It lets you browse and watch videos, manage playlists and visit channels.</p>
+    <p>Features:</p>
+    <ul>
+        <li>Invidious API extended integration!</li>
+        <li>Sync your watch progress between Invidious sessions</li>
+        <li>Mini player.</li>
+        <li>Preview video on hover.</li>
+        <li>Sponsorblock built-in.</li>
+        <li>Return YouTube dislikes built-in.</li>
+        <li>DeArrow built-in (With local processing fallback).</li>
+        <li>Video progress tracking and resuming.</li>
+        <li>No ads.</li>
+        <li>No tracking.</li>
+        <li>Light/Dark themes.</li>
+        <li>Custom colour themes.</li>
+        <li>Integrates with Invidious subscriptions, watch history and more.</li>
+        <li>Live stream support.</li>
+        <li>Dash support.</li>
+        <li>Chapters.</li>
+        <li>Audio only mode.</li>
+        <li>Playlists.</li>
+    </ul>
+  </description>
+  <launchable type="desktop-id">us.materialio.Materialious.desktop</launchable>
+  <categories>
+    <category>AudioVideo</category>
+    <category>Video</category>
+  </categories>
+  <url type="homepage">https://materialio.us</url>
+  <url type="bugtracker">https://github.com/Materialious/Materialious/issues</url>
+  <url type="donation">https://github.com/sponsors/WardPearce</url>
+  <project_license>AGPL-3.0</project_license>
+  <developer_name>WardPearce</developer_name>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Popular videos</caption>
+      <image>https://raw.githubusercontent.com/Materialious/Materialious/7abde6b429d49196d3964ee0a3361712a1a4b4c2/previews/home-preview.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Watching a video</caption>
+      <image>https://raw.githubusercontent.com/Materialious/Materialious/7abde6b429d49196d3964ee0a3361712a1a4b4c2/previews/player-preview.png</image>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">moderate</content_attribute>
+    <content_attribute id="social-audio">moderate</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
```

Comment:
releases missing https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#release